elephant/spike_train_correlation.py

```diff
@@ -646,16 +646,23 @@ def cross_correlation_histogram(
     # resolution
     if binned_st1.matrix_rows != 1 or binned_st2.matrix_rows != 1:
         raise ValueError("Spike trains must be one dimensional")
-    if not np.isclose(binned_st1.binsize.simplified.magnitude,
-                      binned_st2.binsize.simplified.magnitude):
+    if not np.isclose(binned_st1.binsize.simplified.item(),
+                      binned_st2.binsize.simplified.item()):
         raise ValueError("Bin sizes must be equal")
 
     binsize = binned_st1.binsize
-    window_min = -binned_st1.num_bins + 1
-    window_max = binned_st2.num_bins - 1
-
-    t_start_shift = (binned_st2.t_start - binned_st1.t_start) / binsize
-    t_start_shift = t_start_shift.simplified.magnitude
+    left_edge_min = -binned_st1.num_bins + 1
+    right_edge_max = binned_st2.num_bins - 1
+
+    t_lags_shift = (binned_st2.t_start - binned_st1.t_start) / binsize
+    t_lags_shift = t_lags_shift.simplified.item()
+    if not np.isclose(t_lags_shift, int(t_lags_shift)):
```

Comment:
```suggestion
    if not np.isclose(t_lags_shift, round(t_lags_shift)):
```
I would rather round here to prevent potential floating point precision issues.